src/stan-users-guide/simple_linear_regression/run.py

```diff
@@ -0,0 +1,35 @@
+""" Example program that generates data, compiles and runs regression.stan"""
+from cmdstanpy import CmdStanModel
+import numpy
+
+
+# Run from command line: Python regression_1.py
+
+alpha = 2.0
+beta = 3.0
+sigma = 5.0
+n = 100
+x = numpy.random.uniform(size=n)
+y = numpy.random.normal(size=n, loc=alpha + beta * x, scale=sigma)
+
+output = "generating parameters are: alpha={:.1f}, beta={:.1f}, sigma={:.1f}, n={:d}" 
+
+print(output.format(alpha,beta,sigma,n))
+
+stan_data = {'N': n, 'x': x, 'y': y}
+
+stan_program = CmdStanModel(stan_file='regression_naive.stan')
+stan_program.compile()
+fit = stan_program.sample(data=stan_data, output_dir='output')
```

Comment:
Is output the default?  If so, just use that.

Reply:
It puts the files in a tmp dir and deletes them one the python process ends. Not sure what the right thing to do is since the output files have different information than what goes to disk.